src/Controller/ProjectsController.php

```diff
@@ -29,6 +30,7 @@ public function list(ProjectRepository $repository, Request $request, ShuttleRep
         $publishedAtTo = $querypublishedAtTo ? new \DateTime($querypublishedAtTo) : null;
         $state = $request->query->get('projectState');
         $shuttle = $request->query->get('shuttle') ? $shuttleRepository->find($request->query->get('shuttle')) : null;
+        $users = !in_array('', $request->query->all('users')) ? new ArrayCollection($userRepository->findBy(array('id' => $request->query->all('users')))) : null;
```

Comment:
Can't we use `empty` instead ? 
```suggestion
        $users = !empty('', $request->query->all('users')) ? new ArrayCollection($userRepository->findBy(array('id' => $request->query->all('users')))) : null;
```

Reply:
No because `empty($request->query->all('users'))` returns always false as there is an array [0 => ""] when the user inputs manually no members on url like the unit tests "&users%5B%5D="